yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/YamlTestExtension.java

```diff
@@ -58,21 +62,31 @@ public class YamlTestExtension implements TestTemplateInvocationContextProvider,
     private static final Logger logger = LogManager.getLogger(YamlTestExtension.class);
     private List<YamlTestConfig> testConfigs;
     private List<YamlTestConfig> maintainConfigs;
+    private List<ExternalServer> servers;
 
     @Override
     public void beforeAll(final ExtensionContext context) throws Exception {
         if (Boolean.parseBoolean(System.getProperty("tests.runQuick", "false"))) {
             testConfigs = List.of(new EmbeddedConfig());
             maintainConfigs = List.of();
         } else {
-            testConfigs = List.of(
-                    new EmbeddedConfig(),
-                    new JDBCInProcessConfig(),
-                    new MultiServerConfig(0, 1111, 1112),
-                    new ForceContinuations(new MultiServerConfig(0, 1113, 1114)),
-                    new MultiServerConfig(1, 1115, 1116),
-                    new ForceContinuations(new MultiServerConfig(1, 1117, 1118))
-            );
+            AtomicInteger serverPort = new AtomicInteger(1111);
+            List<File> jars = ExternalServer.getAvailableServers();
+            servers = jars.stream().map(jar -> new ExternalServer(jar, serverPort.getAndIncrement(), serverPort.getAndIncrement())).collect(Collectors.toList());
```

Comment:
Just to be safe, I feel like it would be good to have this fail if `jars` is empty.

Reply:
Yeah, good point. Done.